fusionSuite/backend
# Master+organizations users+roles permissions

Changes proposed in this pull request:

- Add roles management
- assign roles to users
- manage permissions on data (type of items and properties)
- manage permissions on structure (types and properties)
- update order of fill data when install (in folder `db/migrations/`)

How to test the feature manually:

1. create a role
2. adjust the permissions
3. assign the role to this user
4. get token for this user
5. do request on endpoints not have access and see 401 return code with the error message

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._

Related to #
